packages/client/src/StreamrClient.ts

```diff
@@ -99,6 +114,15 @@ export class StreamrClient {
         return convertStreamMessageToMessage(result)
     }
 
+    /**
+     * Manually update the current encryption key of a given stream.
+     *
+     * @remarks Only affects the encryption key used when acting as publisher.
```

Comment:
Could maybe combine the description and remarks here? 

Actually it doesn't update the _current_ encryption key if rotate method is used, but instead it queues the key to be used after one message has been published (i.e. the message which uses the previous key and delivered this key inside the `nextGroupKey` field).